addon-test-support/ember-mocha/index.js

```diff
@@ -14,11 +14,20 @@ import {
   TestModuleForAcceptance
 } from 'ember-test-helpers';
 
-const setupTest = setupTestFactory(TestModule);
+const setupTestOld = setupTestFactory(TestModule);
```

Comment:
can you make sure to export `setupTestOld()` too? just in case people are actually using this without a string as first arg

Reply:
Done. Called it `setupTestLegacy`, agree?

Reply:
we're sort of running into a trap here since the old-old thing is already called old already, but I'm okay with that for now... 🙈

Reply:
LOL